src/plugin.js

```diff
@@ -176,15 +176,13 @@ const onPlayerReady = (player, options) => {
     error = videojs.mergeOptions(error, options.errors[error.code || 0]);
 
     if (error.message) {
-      /** This check is made because we want to check if we have got a custom error
-       * message and we do not want to change that
-       */
-      let errmsg = 'The media could not be loaded, either because the server or network' +
-       ' failed or because the format is not supported.';
+      // This check is made because we want to check if we have got a custom error
+      // message and we do not want to change that
+      let custErrmsg = isCustomError;
```

Comment:
We avoid using abbreviations in variable names unless the abbreviation is more widely used than the actual term (e.g. "http"). Also, this variable is not properly camel-cased. Also, this variable is unnecessary since it's exactly the same as `isCustomError`.